docs/manual/developer_guide.adoc

```diff
@@ -969,6 +969,39 @@ This is an example of a check, written using the custom OVAL syntax, that checks
   </unix:file_object>
 ----
 
+===== Macros
+
+Jinja macros for OVAL checks are located in `/shared/macros-oval.jinja`. These currently include the following high-level macros:
```

Comment:
I think we should refer to project directories without `/` in the beginning, because that signalizes an absolute path.

Reply:
I noticed that `macros-ansible.jinja` file is referred to with `/shared`, we should fix them.

Reply:
Ah sorry. I used absolute paths to refer to locations from the root of the `ComplianceAsCode/content` directory, wherever that is for the user. We can use relative locations if you think that's better though.

Reply:
@yuumasato I changed to reference the file using `link`. What do you think?

Reply:
Better than my suggestion.